vm/src/slots.rs

```diff
@@ -223,7 +223,11 @@ where
     T: Unhashable,
 {
     fn hash(_zelf: &PyRef<Self>, vm: &VirtualMachine) -> PyResult<PyHash> {
-        Err(vm.new_type_error("unhashable type".to_owned()))
+        let msg = format!(
+            "unhashable type: '{}'",
+            _zelf.class().name
+        );
+        Err(vm.new_type_error(msg))
```

Comment:
nitpick: you can move format directly in the call here.

Reply:
Do you mean like:

```suggestion
        Err(vm.new_type_error(!format("unhashable type: '{}'", _zelf.class().name)))
```

Reply:
Yup!

Reply:
I applied it in a1d958b commit! 😉